src/EA.Weee.Web/Areas/AATFReturn/Views/SentOnCreateSiteOperator/Index.cshtml

```diff
@@ -0,0 +1,112 @@
+}
+
+<section id="main-content">
+    <h1 class="govuk-heading-l">Which operator was the WEEE sent to for treatment?</h1>
+    @using (Html.BeginRouteForm(AatfRedirect.AatfOrganisationSelectedRoute, FormMethod.Get))
+    {
+        @Html.HiddenFor(m => m.WeeeSentOnId)
+        @Html.HiddenFor(m => m.OrganisationId)
+        @Html.HiddenFor(m => m.AatfId)
+        @Html.HiddenFor(m => m.ReturnId)
+        @Html.Hidden("javascriptDisabled", true)
+
+        <div class="govuk-form-group" id="operatorButton" style="display: none">
+            <button type="submit" class="govuk-button" formmethod="get" name="operatorButtonName">
+                The operator address is the same as the AATF/ATF address
+            </button>
+        </div>
+    }
+    @using (Html.BeginRouteForm(AatfRedirect.AatfOrganisationSelectedRoute, FormMethod.Post))
+    {
+        @Html.AntiForgeryToken()
+        @Html.Gds().ValidationSummary()
+        @Html.HiddenFor(m => m.WeeeSentOnId)
+        @Html.HiddenFor(m => m.SiteAddressData.Address1)
+        @Html.HiddenFor(m => m.SiteAddressData.Address2)
+        @Html.HiddenFor(m => m.SiteAddressData.CountyOrRegion)
+        @Html.HiddenFor(m => m.SiteAddressData.TownOrCity)
+        @Html.HiddenFor(m => m.SiteAddressData.Postcode)
+        @Html.HiddenFor(m => m.SiteAddressData.CountryId)
+        @Html.HiddenFor(m => m.SiteAddressData.CountryName)
+
+        <div class="govuk-form-group" id="operatorCheckBox">
+            @Html.Gds().CheckBoxFrontEndFor(m => m.IsOperatorTheSameAsAATF, new { @class = "govuk-checkboxes__input" })
+            <!--<p class="govuk-body" style="color: grey">If the operator address associated with this AATF/ATF is different please uncheck this box</p>-->
+        </div>
+
+        @Html.EditorFor(m => m.OperatorAddressData)
+
+        <div class="govuk-!-padding-bottom-3"></div>
+        <button type="submit" name="save" class="govuk-button" id="site-details">
+            Save and continue
+        </button>
+
+    }
+</section>
+
+@section scripts {
+    <script>
+        $('#IsOperatorTheSameAsAATF').click(function () {
+            if (this.checked) {
+                var address1 = $('#SiteAddressData_Address1').val();
+                var address2 = $('#SiteAddressData_Address2').val();
+                var townorcity = $('#SiteAddressData_TownOrCity').val();
+                var postcode = $('#SiteAddressData_Postcode').val();
+                var countyorregion = $('#SiteAddressData_CountyOrRegion').val();
+                var countryId = $('#SiteAddressData_CountryId').val();
+                var countryName = $('#SiteAddressData_CountryName').val();
+
+                if (address1 != '') {
+                    $('#OperatorAddressData_Address1').val(address1);
+                }
+
+                if (address2 != '') {
+                    $('#OperatorAddressData_Address2').val(address2);
+                }
+
+                if (townorcity != '') {
```

Comment:
Put this javascript into the sent-on-operator-address.js file and include on this page so it gets minified